code/obj/machinery/interdictor.dm

```diff
@@ -14,19 +14,26 @@
 	var/chargerate = 700 // internal cell charge rate, per tick
 	var/connected = 0 //whether this is tied into a wire
 	var/maglock_cooldown = 3 SECONDS
+	var/emagged = 0 // emagging axes access requirement
 
 	var/canInterdict = 0 // indication of operability
 	//if 0, whether from depletion or new installation, battery charge must reach 100% to set to 1 and activate interdiction
 
 	var/cumulative_cost = 0 // keeps a tally of used power per tick
 	//used to play interdiction noise / modulate its volume
 
+	var/radstorm_paid = FALSE // set during the special radstorm interdiction proc
+	//when true, a cost has already been paid for this process, letting radstorm interdict proc know it doesn't need to expend cell charge again
+
 	var/interdict_range = 5 // range of the interdictor's field
 	//for effects that are wide-band interdicted, such as solar flares, this should dictate the response strength
 
 	var/interdict_class = ITDR_STANDARD // type of interdictor
 	//standard interdictors provide only the stellar phenomena protection; alternate variants unlock new functionality
 
+	var/interdict_cost_mult = 1 // interdiction cost multiplier
+	//some part selections can influence this value, raising or lowing the effective energy cost of interdiction activity
```

Comment:
doc comments

Reply:
Done in [a2c5b2c](https://github.com/goonstation/goonstation/pull/12687/commits/a2c5b2c4369795b21aa94d02747fa63a0b8d19a4).